packages/schemas/src/seeds/scope.ts

```diff
@@ -0,0 +1,11 @@
+import type { CreateScope } from '../db-entries/index.js';
+import { managementResourceId } from './resource.js';
+
+export const managementResourceScopeId = 'management-api-scope';
+
+export const managementResourceScope: Readonly<CreateScope> = Object.freeze({
+  id: managementResourceScopeId,
+  name: 'general',
```

Comment:
how about just `default`?
```suggestion
  name: 'default',
```